index.html

```diff
@@ -237,16 +237,16 @@ <h1>Let's chat</h1>
                 <div class="chat-desc">
                     <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
                 </div>
-                <form action="https://formspree.io/f/xbjbrdyg" method="post" id="form">
+                <form action="https://formspree.io/f/xbjbrdyg" method="post" id="form" onblur="saveInput();">
```

Comment:
- [ ] [OPTIONAL] You should implement this milestone with pair-programming in mind, kindly pay attention to this on the future